pkg/services/live/live.go

```diff
@@ -14,8 +14,9 @@ import (
 )
 
 var (
-	logger   = log.New("live")
-	loggerCF = log.New("live.centrifuge")
+	logger             = log.New("live")
+	loggerCF           = log.New("live.centrifuge")
+	dsHandlerProviders = map[string]models.ChannelHandlerProvider{}
```

Comment:
Can this be an attribute of GrafanaLive instead? Global variables are generally best avoided.